Mathlib/AlgebraicGeometry/EllipticCurve/Jacobian.lean

```diff
@@ -172,7 +172,8 @@ lemma smul_eq (P : Fin 3 → R) {u : R} (hu : IsUnit u) : (⟦u • P⟧ : Point
 
 lemma smul_equiv_smul (P Q : Fin 3 → R) {u v : R} (hu : IsUnit u) (hv : IsUnit v) :
     u • P ≈ v • Q ↔ P ≈ Q := by
-  erw [← Quotient.eq_iff_equiv, ← Quotient.eq_iff_equiv, smul_eq P hu, smul_eq Q hv]
+  rw [← Quotient.eq_iff_equiv, ← Quotient.eq_iff_equiv]
+  erw [smul_eq P hu, smul_eq Q hv]
```

Comment:
Did you try `erw?` here to see if there was any help? If the `erw` is difficult to get rid of, could you add a comment explaining why?

(I was talking with Kim a few hours ago, and they decided that for issues like `![1, 2, 3] 0` not reducing to `1`, which happens a lot for the elliptic curve library apparently, keeping `erw` is fine.)

Reply:
Kim fixed this one already :)